content/get-started/01_get-started.en.md

```diff
@@ -13,25 +29,122 @@ If you want to install Cerebral into an existing project please...
 If you want an introduction you can keep reading and go through a small demo project. Just make sure you fulfill these requirements:
 
 * You have installed [Node](https://nodejs.org/en/) version 4 or later
-* You are okay with learning Cerebral using React and JSX. You can choose something else for your own project later though
+* You have installed [Git](https://git-scm.com/)
+* You are okay with learning Cerebral using React and JSX. You can choose a different Viewlayer (like [Inferno](http://infernojs.org//)) for your own project later though. 
 * You have a coffee or a beer, ready to learn something new
 
-Okay, lets get going. First you need to clone the tutorial repo.
 
-`git clone https://github.com/cerebral/cerebral-website-tutorial.git`
+### Installing the project
 
-Or use [Github Desktop](https://desktop.github.com/)
+Okay, lets get going. First you need to clone cerebral repo. So fire up your command line go to your drive and directory of choice and do:
 
-### Starting the project
-The tutorial runs on [react-create-app](https://facebook.github.io/react/blog/2016/07/22/create-apps-with-no-configuration.html) by Facebook. To run it, go to the project folder and install the dependencies, including Cerebral by running:
+`git clone https://github.com/cerebral/cerebral.git`
 
-`npm install`
+then do a  `cd cerebral/packages/cerebral-website-tutorial`
```

Comment:
Yeah. You are right. Ok then I should just publish latest packages. And it is definetely a way hoe it should work after release.